src/components/pages/ParentHome/Cart.js

```diff
@@ -1,28 +1,29 @@
 import React, { useState, useEffect } from 'react';
 import { connect } from 'react-redux';
-import StripeCheckout from 'react-stripe-checkout';
-import { useHistory } from 'react-router-dom';
+import StripeCheckoutButton from '../Stripe/StripeCheckoutButton';
+// import { useHistory } from 'react-router-dom';
 import { Modal, Button } from 'antd';
 import { timeConverter } from '../../common/timeHelpers';
 import { dateConverter } from '../../common/dateHelpers';
 import {
+  addToCart,
   cancelCartItem,
   clearCart,
 } from '../../../redux/actions/parentActions';
 
 function Cart(props) {
-  const { cart, cancelCartItem, clearCart } = props;
+  const { cart, cancelCartItem } = props; //clearCart needed in deconstructor
   const [showModal, setShowModal] = useState(false);
   let total = cart.reduce((prev, curr) => prev + curr.price, 0);
   const [product, setProduct] = useState({
     name: 'coderheroes',
     price: total,
   });
-  const history = useHistory();
+  // const history = useHistory();
```

Comment:
I think it is okay to leave this and import commented out since we will probably need to use it later for routing

Reply:
Agreed